pkg/version/version.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	ID string = "v0.16.2"
+	ID string = "v0.17.0"
```

Comment:
Could we set this version based on the tags when building the release? So we avoid hardcoding it.

Reply:
We should do that. We're not compiling any binaries as part of the release yet which is a gap in the release process.

Reply:
I don't want to block this release but we should change this soon.